app/views/tag/_topicCard.html.erb

```diff
@@ -32,7 +32,7 @@
       </div>
 
       <div class="card-footer" style="background-color: inherit; border:none;">
-        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= tag.count %> <%= t('tag.index.more_posts') %> &raquo;</a>
+        <a style="padding-top:15px;text-decoration:underline;color:#808080;display:inline-block;" href="/tag/<%= tag.name %>"><%= Tag.counter(tag.name)-shown_nids.count - Tag.find_nodes_by_type(tag.name, type = 'note', limit = 3).where.not(nid: shown_nids).count %> <%= t('tag.index.more_posts') %> &raquo;</a>
```

Comment:
this looks good! I am just wondering why u r subtracting from the `counter` method. Shouldn't all the tags be counted in that method without requiring further operations?

Maybe I am missing something though. What does that subtraction do?

Reply:
Actually this shows "x more posts" on the website so I am subtracting the number of posts that is shown already as you can see in this screenshot 
![image](https://user-images.githubusercontent.com/33183263/76683017-aae2c480-6626-11ea-93f8-96e6b2209345.png)

Reply:
Oh alright thanks. This looks good!